File: src/lib/scala/Slate.Common/src/main/scala/slate/common/logging/LogEntry.scala
/**
<slate_header>
  author: Kishore Reddy
  url: https://github.com/kishorereddy/scala-slate
  copyright: 2015 Kishore Reddy
  license: https://github.com/kishorereddy/scala-slate/blob/master/LICENSE.md
  desc: a scala micro-framework
  usage: Please refer to license on github for more info.
</slate_header>
  */

package slate.common.logging

import java.time.ZoneOffset

import slate.common.DateTime

case class LogEntry(
                      name  : String            = "",
                      level : LogLevel          = Info,
                      msg   : String            = "",
                      time  : DateTime          = DateTime.now,
                      ex    : Option[Exception] = None,
                      tag   : Option[String]    = None
                   )
{
}

File: src/common/common/src/main/kotlin/kiit/common/log/LogEntry.kt
/**
 *  <kiit_header>
 * url: www.kiit.dev
 * git: www.github.com/slatekit/kiit
 * org: www.codehelix.co
 * author: Kishore Reddy
 * copyright: 2016 CodeHelix Solutions Inc.
 * license: refer to website and/or github
 *  </kiit_header>
 */

package kiit.common.log

import kiit.common.DateTime

data class LogEntry(
    val name: String = "",
    val level: LogLevel,
    val msg: String = "",
    val ex: Throwable? = null,
    val tag: String? = null,
    val time: DateTime = DateTime.now()
)